### ISSUE TITLE
Unable to determine R library path error during import

### ISSUE THREAD

**Issue by rockdeme** [rpy2/rpy2#875]
I ran into some issues when trying to import rpy2 in a new virtual environment which was installed using pip on Windows.

`import rpy2` results in the following error message:

```
Unable to determine R library path: Command '('C:\\Users\\ME\\anaconda3\\envs\\test\\Lib\\R\\bin\\Rscript', '-e', 'cat(Sys.getenv("LD_LIBRARY_PATH"))')' returned non-zero exit status 1.
```

I tried installing R using conda in the virtual enironment and I also tried installing R manually to C:\R\ resulting in the same error.

When I run `python -m rpy2.situation` I get the following error:

```
>(test) C:\Users\ME>python -m rpy2.situation

rpy2 version:
3.5.1
Python version:
3.8.13 (default, Mar 28 2022, 06:59:08) [MSC v.1916 64 bit (AMD64)]
Looking for R's HOME:
    Environment variable R_HOME: C:\Users\ME\anaconda3\envs\test\Lib\R
rpy2.situation: Unable to determine R home.
    InstallPath in the registry: None
    Environment variable R_USER: None
    Environment variable R_LIBS_USER: None
    Warning: There is no R in the PATH and no R_HOME defined.
Unable to determine R library path: Command '('C:\\Users\\ME\\anaconda3\\envs\\test\\Lib\\R\\bin\\Rscript', '-e', 'cat(Sys.getenv("LD_LIBRARY_PATH"))')' returned non-zero exit status 1.
R version:
    In the PATH: R version 3.6.1 (2019-07-05) -- "Action of the Toes"
Copyright (C) 2019 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under the terms of the
GNU General Public License versions 2 or 3.
For more information about these matters see
https://www.gnu.org/licenses/.
    Loading R library from rpy2: OK
Additional directories to load R packages from:
None
C extension compilation:
'sh' is not recognized as an internal or external command,
operable program or batch file.
    Warning: Unable to get R compilation flags.
```

I added `C:\Users\ME\anaconda3\envs\test\Lib\R` as R_HOME to the user environment variables and to user PATH as well with no success. I assume that I have to set up the remaining variables as well but I haven't found any info on what directory these variables should point to.

EDIT:

Running the same command using git bash produces the following error:

```
$ python -m rpy2.situation
rpy2.situation: Unable to determine R home.
Unable to determine R library path: Command '('C:\\Users\\ME\\anaconda3\\envs\\test\\Lib\\R\\bin\\Rscript', '-e', 'cat(Sys.getenv("LD_LIBRARY_PATH"))')' returned non-zero exit status 1.
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 179: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 180: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 181: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 182: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 183: make: command not found
R was not built as a library
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 179: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 180: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 181: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 182: make: command not found
C:\Users\ME\anaconda3\envs\test\Lib\R/bin/config.sh: line 183: make: command not found
R was not built as a library
C:\users\ME\Desktop\rpy2-master\rpy2\situation.py:290: UserWarning: No libraries as -l arguments to the compiler.
  warnings.warn('No libraries as -l arguments to the compiler.')
C:\users\ME\Desktop\rpy2-master\rpy2\situation.py:277: UserWarning: No include specified
  warnings.warn('No include specified')
rpy2 version:
3.5.1
Python version:
3.8.13 (default, Mar 28 2022, 06:59:08) [MSC v.1916 64 bit (AMD64)]
Looking for R's HOME:
    Environment variable R_HOME: C:\Users\ME\anaconda3\envs\test\Lib\R
    InstallPath in the registry: None
    Environment variable R_USER: None
    Environment variable R_LIBS_USER: None
    Warning: There is no R in the PATH and no R_HOME defined.
R version:
    In the PATH: R version 3.6.1 (2019-07-05) -- "Action of the Toes"
Copyright (C) 2019 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under the terms of the
GNU General Public License versions 2 or 3.
For more information about these matters see
https://www.gnu.org/licenses/.
    Loading R library from rpy2: OK
Additional directories to load R packages from:
None
C extension compilation:
  include:
  []
  libraries:
  []
  library_dirs:
  []
  extra_compile_args:
  []
  extra_link_args:
  []
(test)
```

**Comment by lgautier** [comment]
Unfortunately `rpy2` is partially supported (and cyclically broken) on Windows, unless you install and use it through WSL (Windows Subsystem for Linux).

**Comment by lgautier** [comment]
Presumed to be fixed with #889. Reopen if needed.
